From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Matti=20Lehtim=C3=A4ki?= <matti.lehtimaki@jolla.com>
Date: Sat, 27 Aug 2022 17:15:03 +0300
Subject: [PATCH] backport: tracker-extract-libav: Add missing include

Nothing was used from tracker-utils.h so replace it with missing
tracker-file-utils.h. Missing header was causing a build warning
and crashing the tracker-extract process.
---
 src/tracker-extract/tracker-extract-libav.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tracker-extract/tracker-extract-libav.c b/src/tracker-extract/tracker-extract-libav.c
index 7c1cd3fb7a1953f8912d86deb3e021b520cbd11c..c5cfaae6df22281ec9d2df980bbda14a48be0074 100644
--- a/src/tracker-extract/tracker-extract-libav.c
+++ b/src/tracker-extract/tracker-extract-libav.c
@@ -22,7 +22,7 @@
 #include <glib.h>
 
 #include <libtracker-sparql/tracker-ontologies.h>
-#include <libtracker-miners-common/tracker-utils.h>
+#include <libtracker-miners-common/tracker-file-utils.h>
 
 #include <libtracker-extract/tracker-extract.h>
 
